workflow-job

```diff
@@ -339,7 +342,7 @@ flows["${sha1}"] = {
     stage "Build"
     node('remote') {
         ws("/home/jenkins/build/workspace/${env.BUILD_NUMBER}") {
-		def integrationArguments = "${dockerIp} \$JAMES_PORT ${sha1ToBuild}"
+		def integrationArguments = "\$JAMES_IP 143 ${sha1ToBuild}"
```

Comment:
here you are using the exported value. Are you sure this is the one you expect?

Reply:
Well, it works. I did not intend to fix that, just to make the damn validation deployment thing work...